src/context/WidgetSettingContext.tsx

```diff
@@ -82,7 +83,7 @@ export const WidgetSettingProvider = ({
       botId,
     });
 
-    const reuseCachedSession = ((
+    const useCachedSession = ((
```

Comment:
Question on:

// However, since the existing logic has been regenerating the channel every 30 days due to this logic.

Is this for backward compat?

Reply:
Rather than backward compat, we could remove the logic that checks the expiration date.
However, since a new channel is created after a fairly long period of 30 days, I believe it is quite reasonable to keep it as is.

Reply:
hmm unless there is a client that come back to channel after 1 year and see the history. There is a way for this? then its ok I think

Reply:
let me ask for opinions on the channel!

Reply:
Note: keep 30 days